### ISSUE TITLE
Brand new to coding etc

### ISSUE THREAD

**Issue by tingletoys** [DidierRLopes/GamestonkTerminal#59]
So, this may not be the right place for this but I am completely new to coding (although not trading). I have downloaded python and the files for this terminal however am having a hell of a time opening and running it. 

Any help on how to install the terminal would be really welcome.

**Comment by RayWillett** [comment]
@tingletoys  can you tell us what you've done to run it? Glancing at the code, it looks like running `python gamestonk_terminal.py` should execute the terminal, but there are a few prerequisite steps if you haven't already handled them

**Comment by DidierRLopes** [comment]
Hey @tingletoys,

I'm dedicating this weekend to make sure that this process of installation is drastically improved. Just bear with me as I had to learn some new things along the way. :)

**Comment by joaomotap** [comment]
I wasn't able to run it either, got an "illegal hardware instruction" error. First time I've seen it on a python script. Maybe something in there doesn't play well with the M1 cpu...?

**Comment by DidierRLopes** [comment]
> 
> 
> I wasn't able to run it either, got an "illegal hardware instruction" error. First time I've seen it on a python script. Maybe something in there doesn't play well with the M1 cpu...?

Hey Joao,

I've never seen that either. But from a quick search online it appears to be related with tensorflow. Have a look at this thread https://github.com/apple/tensorflow_macos/issues/143

Hope it helps!

**Comment by barbar055a** [comment]
Hi Didier, it works very well, but how do I use the outputs from the 'sen' section? Some commands such as 'mentions' have no output, while others such as 'infer' outputs a .pt file.

**Comment by tmdickinson** [comment]
> 
> 
> I wasn't able to run it either, got an "illegal hardware instruction" error. First time I've seen it on a python script. Maybe something in there doesn't play well with the M1 cpu...?

Try opening up an interactive python interpreter and just try to import flair and tensorflow. I was getting "illegal instruction" errors too and was able to trace it down to those two imports. I commented out those imports (flair is imported by sentiment/twitter_api.py and tensorflow is imported in prediction_techniques/neural_networks.py) and I was able to get everything else to run.
